.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,54 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '45 1 * * 0'
+  push:
```

Comment:
I am curious why don't we enforce the check on pull_request? Shouldn't the check beforehand is more important than afterwards?

Reply:
The scorecard checks does not look to the code for vulnerability, this would be the job for a SAST tool for example. Thus, the scorecard running on pull request would not be a significant addition to the pull request process. The checks are more related to the tools and configurations you use in the project.